html/css/panel.css

```diff
@@ -483,4 +483,20 @@ input.search:focus  {
 
 .download-text p {
 
+}
+
+
+#reset-list-button-div,
+#undo-list-button-div{
+  text-align: right;
+}
+
+.unfiltered-list-message{
```

Comment:
It makes sense to use existing classes. You have used the generic 'alert-warning' so you could use 'alert-success' as well for you unfiltered list message